docs/user-guide/alert-manager.rst

```diff
@@ -52,6 +52,11 @@ After you configured the ``receiver`` and ``route``, you can test it works prope
 
 You should see the demo alert, in the Robusta UI, Slack, or any other configured ``sink`` within a few minutes
 
+.. admonition:: "Alerts won't show up" UI notification
+    :class: warning
+
+    "Alerts won't show up because Prometheus is not connected to Robusta" will always be there even if the connection is configured correctly, until the first alert has been raised.
```

Comment:
I would try to make it shorter. How about:
The notification is displayed until the first alert is sent from AlertManager to Robusta

Reply:
Made the specified changes.